frontends/precinct-scanner/src/screens/polls_closed_screen.test.tsx

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import { render, screen } from '@testing-library/react';
+import { Inserted } from '@votingworks/test-utils';
+import { electionSampleDefinition } from '@votingworks/fixtures';
+import { AppContext } from '../contexts/app_context';
+import { PollsClosedScreen } from './polls_closed_screen';
+
+const TEST_BALLOT_COUNT = 50;
+const MACHINE_ID = '0003';
+
+function renderScreen(showNoChargerWarning: boolean) {
```

Comment:
Maybe change this to take `props: Partial<PollsClosedScreenProps> = {}` as its argument and then call like `renderScreen({ showNoChargerWarning: false })`. This is a pattern I've seen us using elsewhere. Otherwise it's hard to know what the boolean means just from looking at the call site of `renderScreen`.

Reply:
We love some well-targeted Typescript, thanks for the suggestion.